rapid7_insightvm/plugin.spec.yaml

```diff
@@ -4,7 +4,7 @@ products: [insightconnect]
 name: rapid7_insightvm
 title: Rapid7 InsightVM
 description: InsightVM is a powerful vulnerability management tool which finds, prioritizes, and remediates vulnerabilities. This plugin can get top remediations, scan results and start scans
-version: 4.2.0
+version: 4.2.1
```

Comment:
```suggestion
version: 5.0.0
```
Needs a major bump as this will break any action configured in a workflow that does not have the Description input.